API/api-gestion.yml

```diff
@@ -570,3 +570,140 @@ paths:
                     
 #########################################################################################
 
+  /soporte:
+	# Creacion de una ayuda o ticket al soporte tecnico
+    post:
+      summary: Consulta una petición a soporte tecnico
+      tags:
+        - Soporte tecnico
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - id_usuario
+                - descripcion_problema
+                - fecha_problema
+              properties:
+                id_usuario:
+                  type: integer
+                descripcion_problema:
+                  type: string
+                fecha_problema:
+                  type: string
+            example:
+              id_usuario: 1
+              fecha_problema: 20/04/2022
+              descripcion_problema: Problema con la aplicación movil
+      responses:
+        '200':
+          description: Peticion correcta
+          content:
+            application/json:
+              examples:
+                Insertar petición a soporte tecnico:
+                  value: |-
+                    {
+                      "id_usuario": "001",
+                      "fecha_problema": "20/04/2022",
+                      "descripcion_problema": "Problema con la aplicación móvil"
+                    }
+        '400':
+          description: Peticion incorrecta
+          content:
+            application/json:
+              examples:
+                Error al meter incidencia:
+                  value: '{"code": "400", "info" :"Malformed request syntax."}'
+  #Cogemos toda informacion sobre los tickes de un usuario que ha pedido ayuda al soporte tecnico
+  /soporte/userid:
```

Comment:
Entocnes aqui en vez de usar userid usaria ticket id, ya que un usuario puede tener mas de un ticketm y entiendo que este get es para ver el ticket en si


![image](https://user-images.githubusercontent.com/22755318/164458799-49a38a6f-0604-43f5-8d25-be2f4a7614c8.png)

Reply:
y para la parte del delete igual, es mejor eliminar ticket_id que tickets de usuarios

Reply:
y tambien para que la web sea capaz de pasar el  ticket el formato debe ser algo tal que asi:

  /soporte/{id}: # <------ fijaros que el id tiene unos brakets y despues en parameters se dice que este id es un int
      get:
        tags: 
        - Trabajadores
        parameters:
        - in: path
          name: id
          required: true
          schema:
            type: integer


Asi el id puede ser un numero y si se va a /soporte/55 iremos al ticket 55